src/components/orders/OrdersUserDetailsTable/index.tsx

```diff
@@ -48,41 +88,61 @@ export type Props = StyledUserDetailsTableProps & {
 
 interface RowProps {
   order: Order
-  isPriceInverted: boolean
+  _isPriceInverted: boolean
```

Comment:
I would prefer to have it the other way around.
Keep externally visible prop without `_` and if you need change the name internally